src/bindings/overlay.js

```diff
@@ -10,6 +10,7 @@ const allowed = {
   ],
   attributes: {
     global: ['title', 'aria-label', 'aria-valuetext', 'aria-moz-hint'],
+    button: ['accesskey'],
```

Comment:
Yeah, I'm not opposed but also don't want to optimize dir layout too early.  I'm okay with having this `button` here even for HTML for now.